docs/api.yml

```diff
@@ -0,0 +1,307 @@
+openapi: "3.0.0"
+info:
+  title: RT Dashboard API
+  version: 1.0.0
+
+paths:
+  /history.json:
+    get:
+      summary: List history entries
+      responses:
+        200:
+          description: |
+            Return object containting list of rows and groups of history entries
+          content:
+            application/json:
+              schema:
+                type: object
+                required:
+                  - rows
+                  - groups
+                properties:
+                  rows:
+                    type: array
+                    items:
+                      type: object
+                      required:
+                        - group
+                        - subgroup
+                        - start
+                        - title
+                        - type
+                        - content
+                      properties:
+                        group:
+                          type: string
+                          description: Name of the task
+                        subgroup:
+                          type: integer
+                          description: Id of the worker
+                        start:
+                          type: integer
+                          format: timestamp
+                          description: Start time of the task in unix timestamp format
+                        end:
+                          type: integer
+                          format: timestamp
+                          description: End time of the task in unix timestamp format when `type` is range
+                        type:
+                          type: string
+                          enum: [range, point]
+                        content:
+                          type: string
+                          description: Short info about the task
```

Comment:
Info about the worker the task is running on

Reply:
```suggestion
                          description: |
                            

Info about the worker the task is running on, or a time range string, or the starting time string, depending on the state
```